CHANGELOG.md

```diff
@@ -5,6 +5,10 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 
 ## [Unreleased]
 
+### Changed
+
+- Switch to Java 11 for build and run
```

Comment:
Please tell that we still compile to target JDK 8

Reply:
That's not the case. This PR switch everything to java 11